### PR TITLE
feat: estimate the version from the hash

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Uses hashes to guess the version of the installed program. This makes it easier for people who already have AviUtl installed to use apm.

**This PR depends on the following PR(s).**
- https://github.com/hal-shu-sato/apm-data/pull/43

## How Has This Been Tested?

https://github.com/hal-shu-sato/apm-data/pull/43 must be loaded in apm.
- If you start apm after deleting apm.json, the version will still be displayed.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by hal-shu-sato** [comment]
I suggest changing the feature like SRI.
I think that naming the attribute `integrity` and using the `ssri` package would make it more versatile.
https://developer.mozilla.org/ja/docs/Web/Security/Subresource_Integrity

**Comment by mitosagi** [comment]
Using SRI is a good idea. I have also modified apm-data.

**Review by hal-shu-sato — COMMENTED** [review]
Do you have `yarn.lock`, in which `ssri` is installed?

**Comment by mitosagi** [comment]
There was no diff in `yarn.lock`. It seems to be in there already.

**Comment by hal-shu-sato** [comment]
Oh, that's right.
Sorry.

File: src/lib/parseXML.js [hunk 1]
```diff
@@ -102,7 +102,12 @@ class CoreInfo {
       this.releases = {};
       const prefix = parsedCore.releases[0].$prefix[0];
       for (const fileURL of parsedCore.releases[0].fileURL) {
-        this.releases[fileURL.$version[0]] = path.join(prefix, fileURL._[0]);
+        const release = [];
```
Comment (hal-shu-sato) [inline src/lib/parseXML.js:105]: Is this an object?
```suggestion
        const release = {};
```

**Review by hal-shu-sato — COMMENTED** [review]
I have a point to check.

**Comment by mitosagi** [comment]
I have fixed it.

**Review by hal-shu-sato — APPROVED** [review]
Good feature.

**Pull request merged**